starter-code/index.html

```diff
@@ -120,17 +120,21 @@
     <script src="/vendor/scripts/webdb.js"></script>
     <script src="/vendor/scripts/page.js"></script>
 
-    <!-- TODO: Include your secret GitHub token file, which should be gitingore'd,
+    <!-- TODO:IGNORE Include your secret GitHub token file, which should be gitingore'd,
       therefore recreated in every developer's environment. -->
+    <script src="scripts/models/githubToken.js"></script>
```

Comment:
You may want to leave this in the root directory since it's technically not a model. Up to you though for the overall design.